chains/orchestrator-relays/runtime/dancelight/src/bridge_to_ethereum_config.rs

```diff
@@ -345,6 +333,272 @@ where
     }
 }
 
+/// `NativeContainerTokensProcessor` is responsible for receiving and processing native container
+/// chain tokens coming from Ethereum and forwarding them to the container chain via Tanssi through XCM.
+pub struct NativeContainerTokensProcessor<
```

Comment:
why is this in the runtime itself if the other processor is in common? We should anticipate we will need it in starlight and therefore put it in common if possible